include/RAJA/pattern/tensor/TensorBlock.hpp

```diff
@@ -0,0 +1,366 @@
+/*!
+ ******************************************************************************
+ *
+ * \file
+ *
+ * \brief   RAJA header file defining SIMD/SIMT register operations.
+ *
+ ******************************************************************************
+ */
+
+//~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~//
+// Copyright (c) 2016-22, Lawrence Livermore National Security, LLC
+// and RAJA project contributors. See the RAJA/LICENSE file for details.
+//
+// SPDX-License-Identifier: (BSD-3-Clause)
+//~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~//
+
+#if 0
```

Comment:
Is this file something we want to keep around @ajkunen?

Reply:
yeah, that will get reenabled with a future PR... i suppose i should have removed it from this PR, but it's already merged :(

Reply:
🤷 not a big deal, just noticed it while reading through and it had me wondering.  Awesome work on this whole thing.